### DISCUSSION TITLE
Give a time value to the way points

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#140] [Ideas]
I have seen the "Add editable way-points between unit positions" I tried it a bit, and I ask myself if it could be possible to add a waypoint and then assign it the time when the unit passed on it.
Thank you
Federico

**Comment by kjellmf** [comment]
Thank you for the suggestion. I have considered this and will probably implement something like that. I have not settled yet on the best way to edit the trajectory of a unit. The current approach is nice for sketching the route to avoid for instance that a ship travels over land.

**Comment by federicopeyrani1969** [comment]
HI Kjell, I am now trying to edit time stamps for way points, but I can't find the way to edit them (maybe you already told me?).
Is it possible? I seem to remember it was possible once. There was a table with time stamps for each unit, am I wrong?

**Reply by federicopeyrani1969** [reply]
Ooooops! I found it in unit state! Sorry for bothering you!
